package.json

```diff
@@ -3,7 +3,7 @@
   "version": "11.1.0",
   "description": "Additional ESLint's rules for Node.js",
   "engines": {
-    "node": ">=8.10.0"
+    "node": "^10.12.0 || >=12.0.0"
```

Comment:
Don't changes this. Making this change would mean that you break support for folks using older versions of eslint.

Reply:
The previous tests were only checking older ESLint versions against the latests Node version.

Reply:
Yep sure, the CI improvements are great. But this  line is a breaking change to the runtime of the package - previously folks could use eslint 5 and node 8 and could install this package, and if you change this line they'll get an error.

What happens if you don't make this change? What breaks? Folks can still use this package, eslint 5 with node 8, and they can also use eslint 7 with node 10.